README.md

```diff
@@ -118,7 +118,8 @@ After installing the SGX SDK and SGX SSL double check that ``SGX_SDK`` and
 ``SGX_SSL`` variables are set correctly in your env. In particular,
 the makefiles are configured with meaningful defaults for variables
 like ``SGX_SDK``, ``SGX_ARCH``, ``SGX_MODE``, ``SGX_BUILD`` or
-``SGX_SSL``.  However, if you have non-standard values for install
+``SGX_SSL``. Notice: by default, ``SGX_MODE=HW``.
```

Comment:
Can you also add `SGX_MODE=Sim` as an option here so users know what available options.

Reply:
I have rephrased it though... it was written immediately after...by you!! :)

Reply:
by me ... 🙈